modules/lambdas/variables.tf

```diff
@@ -200,7 +200,16 @@ variable "orca_recovery_retry_interval" {
   description = "Number of seconds to wait between recovery failure retries."
 }
 
+
 variable "orca_recovery_retry_backoff" {
   type        = number
   description = "The multiplier by which the retry interval increases during each attempt."
-}
\ No newline at end of file
+}
+
+
+# Set to Default value of "INFO"
+variable "log_level" {
+  type        = string
+  description = "Sets the verbose of powertools logger"
```

Comment:
Probably should require this variable to be one of the following strings per the docs.
 "INFO", "DEBUG", "WARN", "ERROR", etc.

Reply:
done